samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/ui/component/AddPropertyLayout.kt

```diff
@@ -93,7 +92,7 @@ internal fun AddPropertyLayout.toProperty(): Property {
         is_staging = radio_stage.isChecked,
         targetingParameters = ccpaTp + gdprTp,
         statusCampaignSet = setOf(gdprStatus, ccpaStatus),
-        messageType = message_type_autocomplete.text.toString(),
+        messageType = "SAMPLE_STRING",
```

Comment:
I think we can remove the field messageType from the DTOs, to be more specific, using the function "Refactor-> Change Signature" you can delete this field from 2 classes:
- Property (in the file Model.kt)
- PropertyDTO (in the file Models4View.kt)